### ISSUE TITLE
AWS::AppConfig::Application was erroneously moved in updated CFN specs

### ISSUE THREAD

**Issue by david-dreyer** [aws-cloudformation/cfn-lint#2897]
### CloudFormation Lint Version

v0.80.4

### What operating system are you using?

Ubuntu

### Describe the bug

This latest version updated the CFN Specs for validation (https://github.com/aws-cloudformation/cfn-lint/pull/2889/files#diff-a5a6f378b19287fd419ef6c9edf44d4493163add0a727420af026573a02f165a).

AWS had somehow erroneously moved `AWS::AppConfig::Application` underneath a different resource, and so it was deleted from cfn-lint's model.

AWS has fixed this on their end, but we need a new cfn-lint release that restores this in the cfn specs.

### Expected behavior

AWS::AppConfig::Application should be a valid resource type

### Reproduction template

```yaml
AWSTemplateFormatVersion: 2010-09-09
Description: Reproduce AWS::AppConfig::Application not recognized
Resources:
  AppConfigApplication:
    Type: AWS::AppConfig::Application
    Properties:
      Name: !Join
        - ""
        - - TestApplication
          - !Select
            - 0
            - !Split
              - "-"
              - !Select
                - 2
                - !Split
                  - "/"
                  - !Ref AWS::StackId
```

**Comment by kddejong** [comment]
Added back in #2893 2893
